src/Microsoft.Diagnostics.Runtime.Tests/Microsoft.Diagnostics.Runtime.Tests.csproj

```diff
@@ -4,9 +4,6 @@
     <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
 
-    <!-- This _IS_ a Unit Test Project, but because of build_test_assets.cmd, we don't want to run as part of CI right now -->
```

Comment:
This should be in its own pull request.  Currently the tests fail when this block is removed, even on Linux.  I haven't had a chance to dig through it yet.  I agree we should remove this block, but it should be in its own PR with associated fixes.

Reply:
This isn't part of the PR, but the PR will be blocked until the tests pass.